plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/IcebergTestUtils.java

```diff
@@ -143,4 +146,10 @@ public static boolean checkParquetFileSorting(TrinoInputFile inputFile, String s
         }
         return true;
     }
+
+    public static TrinoFileSystemFactory getFileSystemFactory(DistributedQueryRunner queryRunner)
+    {
+        return ((IcebergConnector) queryRunner.getCoordinator().getConnector(ICEBERG_CATALOG))
```

Comment:
`queryRunner().getDefautlSession().getCatalog()`
 instead of hard-coding the catalog name

Reply:
This is using the constant from `IcebergQueryRunner` that is used to both set the default session catalog and to create the catalog. It's not hard-coded. If we change the behavior of `IcebergQueryRunner`, maybe the session catalog might no longer be the Iceberg catalog? We already use this constant in other places, so this seems more direct.